docs/src/pages/components/data-grid/pagination/SizePaginationGrid.js

```diff
@@ -3,15 +3,26 @@ import { DataGrid } from '@material-ui/data-grid';
 import { useDemoData } from '@material-ui/x-grid-data-generator';
 
 export default function SizePaginationGrid() {
+  const [pageSize, setPageSize] = React.useState(5);
+
   const { data } = useDemoData({
     dataSet: 'Commodity',
     rowLength: 100,
     maxColumns: 6,
   });
 
+  const handleOnPageSizeChange = (params) => {
```

Comment:
The naming convention:

```suggestion
  const handlePageSizeChange = (params) => {
```